packages/ipfs-daemon/src/ipfs-daemon.ts

```diff
@@ -80,6 +84,8 @@ export class IpfsDaemon {
 
         const repo = configuration.ipfsS3RepoEnabled ? createRepo({
             path: configuration.ipfsPath,
+            local: configuration.ipfsLocal,
+            createIfMissing: configuration.ipfsCreateIfMissing
```

Comment:
The default here for `createIfMissing` should be the same as whatever the current default is, which I think is true.